primitives/src/lib.rs

```diff
@@ -63,9 +63,15 @@ pub const DEFAULT_COMMITTEE_SIZE: u32 = 4;
 pub const DEFAULT_BAN_MINIMAL_EXPECTED_PERFORMANCE: Perbill = Perbill::from_percent(0);
 pub const DEFAULT_BAN_SESSION_COUNT_THRESHOLD: SessionCount = 3;
 pub const DEFAULT_BAN_REASON_LENGTH: u32 = 300;
+
 pub const DEFAULT_CLEAN_SESSION_COUNTER_DELAY: SessionCount = 960;
 pub const DEFAULT_BAN_PERIOD: EraIndex = 10;
 
+/// Version returned when no version has been set.
+pub const DEFAULT_FINALITY_VERSION: Version = 0;
+/// Version of the legacy abft.
+pub const LEGACY_FINALITY_VERSION: u16 = 1;
+
```

Comment:
It's perhaps best to define here also `CURRENT_FINALITY_VERSION` and import in `finality-aleph/src/abft/current.rs`

Reply:
Changed.